packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.util.ts

```diff
@@ -0,0 +1,75 @@
+import {assert} from '@webex/test-helper-chai';
+import {anonymizeIPAddress, clearEmptyKeysRecursively, userAgentToString} from '../../../../src/call-diagnostic/call-diagnostic-metrics.util';
+import sinon from 'sinon';
+import * as anonymize from 'ip-anonymize';
+
+
+describe('internal-plugin-metrics', () => {
+  // describe('userAgentToString', () => {
```

Comment:
why is this commented out?

Reply:
I've left some comments on the PR, but i've been struggling mocking modules while using sinon.

Reply:
Arun suggested I should use jest, but I noticed that there is nothing in beta that uses `jest`, everything is on master......

Reply:
Unless you have a solution on how to mock the modules, I've deleted them for now, I can add them back if you know and tell me how to do it.